installscripts/jazz-terraform-unix-noinstances/acl.tf

```diff
@@ -0,0 +1,47 @@
+resource "aws_rds_cluster" "casbin" {
+  cluster_identifier      = "casbin-cluster"
+  availability_zones      = ["us-east-1a", "us-east-1b"]
+  database_name           = "${var.acl_db_name}"
+  master_username         = "${var.acl_db_username}"
+  master_password         = "${var.acl_password}"
+  port                    = "${var.acl_db_port}"
+  backup_retention_period = 5
+  preferred_backup_window = "07:00-09:00"
+  skip_final_snapshot = true
+  engine                  = "aurora-mysql"
+  engine_version          = "5.7.12"
+  vpc_security_group_ids  = ["${aws_security_group.acl_sg.id}"]
+
+  tags = "${merge(var.additional_tags, local.common_tags)}"
+}
+
+resource "aws_rds_cluster_instance" "casbin-instance" {
+  apply_immediately       = true
+  cluster_identifier      = "${aws_rds_cluster.casbin.id}"
+  identifier              = "casbin"
+  instance_class          = "db.t2.small"
+  engine                  = "aurora-mysql"
+  engine_version          = "5.7.12"
+  publicly_accessible     = true
+
+  tags = "${merge(var.additional_tags, local.common_tags)}"
+}
+
+# TO DO  - to remove in next version
```

Comment:
Sorry, I was referring to `cidr_blocks` specifically.  You can say - `#TODO:  CIDR  block will be refined/more restricted in the next version`

Reply:
Done